app/gateway-oss/2.4.x/upgrading.md

```diff
@@ -122,9 +115,9 @@ please migrate to 1.5.x first.
 
 The steps for upgrading from 0.14.1 to 1.5.x are the same as upgrading
 from 0.14.1 to Kong 1.0. Please follow the steps described in the
-"Migration Steps from 0.14" in the 
-[Suggested Upgrade Path for Kong 1.0](#kong-1-0-upgrade-path), 
-with the addition of the `kong migrations migrate-apis` command, 
+"Migration Steps from 0.14" in the
+[Suggested Upgrade Path for Kong 1.0](#kong-1-0-upgrade-path),
+with the addition of the `kong migrations migrate-apis` command,
```

Comment:
I can't leave a suggestion for line 126, but it should be a level 3 heading (### Upgrade from . . .) since the one before it is a level 3 heading. Line 186 should also be a level 3 heading for the same reason.